### ISSUE TITLE
Admin Dashboard Order List Issue

### ISSUE THREAD

**Issue by ricardolopesdev** [medusajs/medusa#9052]
### System information

Medusa version (including plugins):preview
Node.js version:20.17.0
Database:PostgreSQL
Operating system: Ubuntu 22.04.4 
Browser (if relevant):

### Expected behavior
In order detail, the order it's  shipped, but when return back to orders list the order don't change te state. It's need refresh the browser to show the correct state of order.

```
{
  "id": "order_01J7AW16KM9J04DYEWKRF5SJSM",
  "status": "pending",
  "created_at": "2024-09-09T07:47:35.157Z",
  "canceled_at": null,
  "email": "user@test.com",
  "display_id": 3,
  "currency_code": "eur",
  "total": 20,
  "subtotal": 20,
  "shipping_total": 10,
  "shipping_tax_total": 0,
  "tax_total": 0,
  "summary": {
    "paid_total": 20,
    "difference_sum": 0,
    "raw_paid_total": {
      "value": "20",
      "precision": 20
    },
    "refunded_total": 0,
    "transaction_total": 20,
    "pending_difference": 0,
    "raw_difference_sum": {
      "value": "0",
      "precision": 20
    },
    "raw_refunded_total": {
      "value": "0",
      "precision": 20
    },
    "current_order_total": 20,
    "original_order_total": 20,
    "raw_transaction_total": {
      "value": "20",
      "precision": 20
    },
    "raw_pending_difference": {
      "value": "0",
      "precision": 20
    },
    "raw_current_order_total": {
      "value": "20",
      "precision": 20
    },
    "raw_original_order_total": {
      "value": "20",
      "precision": 20
    }
  },
  "version": 3,
  "customer_id": "cus_01J7AW11PHCA62GP0NBYM7DJKB",
  "sales_channel_id": "sc_01J743P6PA41FZ5GF1EYVYFC9Q",
  "items": [
    {
      "id": "ordli_01J7AW16KN9DACFVRNV9QX1Z7K",
      "title": "M",
      "subtitle": "Medusa Shorts",
      "thumbnail": "https://medusa-public-images.s3.eu-west-1.amazonaws.com/shorts-vintage-front.png",
      "variant_id": "variant_01J743P76N4J5WM4BJE09GYZRZ",
      "product_id": "prod_01J743P760PB35PP3VEH719K0A",
      "product_title": "Medusa Shorts",
      "product_description": "Reimagine the feeling of classic shorts. With our cotton shorts, everyday essentials no longer have to be ordinary.",
      "product_subtitle": null,
      "product_type": null,
      "product_collection": null,
      "product_handle": "shorts",
      "variant_sku": "SHORTS-M",
      "variant_barcode": null,
      "variant_title": "M",
      "variant_option_values": null,
      "requires_shipping": true,
      "is_discountable": true,
      "is_tax_inclusive": false,
      "raw_compare_at_unit_price": null,
      "raw_unit_price": {
        "value": "10",
        "precision": 20
      },
      "is_custom_price": false,
      "metadata": {},
      "created_at": "2024-09-09T07:47:35.157Z",
      "updated_at": "2024-09-09T07:47:35.157Z",
      "deleted_at": null,
      "tax_lines": [],
      "adjustments": [],
      "compare_at_unit_price": null,
      "unit_price": 10,
      "quantity": 1,
      "raw_quantity": {
        "value": "1",
        "precision": 20
      },
      "detail": {
        "id": "orditem_01J7AW2FQ65RYT2D2EV6C42738",
        "order_id": "order_01J7AW16KM9J04DYEWKRF5SJSM",
        "version": 3,
        "item_id": "ordli_01J7AW16KN9DACFVRNV9QX1Z7K",
        "raw_quantity": {
          "value": "1",
          "precision": 20
        },
        "raw_fulfilled_quantity": {
          "value": "1",
          "precision": 20
        },
        "raw_shipped_quantity": {
          "value": "1",
          "precision": 20
        },
        "raw_return_requested_quantity": {
          "value": "0",
          "precision": 20
        },
        "raw_return_received_quantity": {
          "value": "0",
          "precision": 20
        },
        "raw_return_dismissed_quantity": {
          "value": "0",
          "precision": 20
        },
        "raw_written_off_quantity": {
          "value": "0",
          "precision": 20
        },
        "metadata": null,
        "created_at": "2024-09-09T07:48:17.258Z",
        "updated_at": "2024-09-09T07:48:17.258Z",
        "deleted_at": null,
        "quantity": 1,
        "fulfilled_quantity": 1,
        "shipped_quantity": 1,
        "return_requested_quantity": 0,
        "return_received_quantity": 0,
        "return_dismissed_quantity": 0,
        "written_off_quantity": 0
      },
      "subtotal": 10,
      "total": 10,
      "original_total": 10,
      "discount_total": 0,
      "discount_tax_total": 0,
      "tax_total": 0,
      "original_tax_total": 0,
      "refundable_total_per_unit": 10,
      "refundable_total": 10,
      "fulfilled_total": 10,
      "shipped_total": 10,
      "return_requested_total": 0,
      "return_received_total": 0,
      "return_dismissed_total": 0,
      "write_off_total": 0,
      "raw_subtotal": {
        "value": "10",
        "precision": 20
      },
      "raw_total": {
        "value": "10",
        "precision": 20
      },
      "raw_original_total": {
        "value": "10",
        "precision": 20
      },
      "raw_discount_total": {
        "value": "0",
        "precision": 20
      },
      "raw_discount_tax_total": {
        "value": "0",
        "precision": 20
      },
      "raw_tax_total": {
        "value": "0",
        "precision": 20
      },
      "raw_original_tax_total": {
        "value": "0",
        "precision": 20
      },
      "raw_refundable_total_per_unit": {
        "value": "10",
        "precision": 20
      },
      "raw_refundable_total": {
        "value": "10",
        "precision": 20
      },
      "raw_fulfilled_total": {
        "value": "10",
        "precision": 20
      },
      "raw_shipped_total": {
        "value": "10",
        "precision": 20
      },
      "raw_return_requested_total": {
        "value": "0",
        "precision": 20
      },
      "raw_return_received_total": {
        "value": "0",
        "precision": 20
      },
      "raw_return_dismissed_total": {
        "value": "0",
        "precision": 20
      },
      "raw_write_off_total": {
        "value": "0",
        "precision": 20
      },
      "variant": {
        "id": "variant_01J743P76N4J5WM4BJE09GYZRZ",
        "title": "M",
        "sku": "SHORTS-M",
        "barcode": null,
        "ean": null,
        "upc": null,
        "allow_backorder": false,
        "manage_inventory": false,
        "hs_code": null,
        "origin_country": null,
        "mid_code": null,
        "material": null,
        "weight": null,
        "length": null,
        "height": null,
        "width": null,
        "metadata": null,
        "variant_rank": 0,
        "product_id": "prod_01J743P760PB35PP3VEH719K0A",
        "product": {
          "id": "prod_01J743P760PB35PP3VEH719K0A",
          "title": "Medusa Shorts",
          "handle": "shorts",
          "subtitle": null,
          "description": "Reimagine the feeling of classic shorts. With our cotton shorts, everyday essentials no longer have to be ordinary.",
          "is_giftcard": false,
          "status": "published",
          "thumbnail": "https://medusa-public-images.s3.eu-west-1.amazonaws.com/shorts-vintage-front.png",
          "weight": "400",
          "length": null,
          "height": null,
          "width": null,
          "origin_country": null,
          "hs_code": null,
          "mid_code": null,
          "material": null,
          "collection_id": null,
          "collection": null,
          "type_id": null,
          "discountable": true,
          "external_id": null,
          "created_at": "2024-09-06T16:46:42.878Z",
          "updated_at": "2024-09-06T16:46:42.878Z",
          "deleted_at": null,
          "metadata": null,
          "type": null
        },
        "created_at": "2024-09-06T16:46:42.901Z",
        "updated_at": "2024-09-06T16:46:42.901Z",
        "deleted_at": null,
        "options": [
          {
            "id": "optval_01J743P767ETV91EQAK450S38V",
            "value": "M",
            "option_id": "opt_01J743P766V3Y75APRN7NHWSQZ",
            "option": {
              "id": "opt_01J743P766V3Y75APRN7NHWSQZ",
              "product_id": null
            },
            "metadata": null,
            "created_at": "2024-09-06T16:46:42.878Z",
            "updated_at": "2024-09-06T16:46:42.878Z",
            "deleted_at": null
          }
        ],
        "inventory_items": []
      }
    }
  ],
  "shipping_address": {
    "id": "caaddr_01J7AW11PXGCVWPBWY2TKY14YW",
    "customer_id": null,
    "company": "Test",
    "first_name": "Test",
    "last_name": "Test",
    "address_1": "Test",
    "address_2": "",
    "city": "Test",
    "country_code": "de",
    "province": "test",
    "postal_code": "Test",
    "phone": "test",
    "metadata": null,
    "created_at": "2024-09-09T07:47:30.141Z",
    "updated_at": "2024-09-09T07:47:30.141Z"
  },
  "billing_address": {
    "id": "caaddr_01J7AW11PXFHGQF2ARDT1HMH6P",
    "customer_id": null,
    "company": "Test",
    "first_name": "Test",
    "last_name": "Test",
    "address_1": "Test",
    "address_2": "",
    "city": "Test",
    "country_code": "de",
    "province": "test",
    "postal_code": "Test",
    "phone": "test",
    "metadata": null,
    "created_at": "2024-09-09T07:47:30.141Z",
    "updated_at": "2024-09-09T07:47:30.141Z"
  },
  "shipping_methods": [
    {
      "id": "ordsm_01J7AW16KMYJGYYKC535M3QSK0",
      "name": "Standard Shipping",
      "description": null,
      "raw_amount": {
        "value": "10",
        "precision": 20
      },
      "is_tax_inclusive": false,
      "is_custom_amount": false,
      "shipping_option_id": "so_01J743P6WB5F22NJ59HAK8W3AW",
      "data": {},
      "metadata": null,
      "created_at": "2024-09-09T07:47:35.157Z",
      "updated_at": "2024-09-09T07:47:35.157Z",
      "deleted_at": null,
      "tax_lines": [],
      "adjustments": [],
      "amount": 10,
      "order_id": "order_01J7AW16KM9J04DYEWKRF5SJSM",
      "detail": {
        "id": "ordspmv_01J7AW2FQ7X8P4FMB6Q6KK72SZ",
        "order_id": "order_01J7AW16KM9J04DYEWKRF5SJSM",
        "version": 3,
        "shipping_method_id": "ordsm_01J7AW16KMYJGYYKC535M3QSK0",
        "created_at": "2024-09-09T07:47:35.157Z",
        "updated_at": "2024-09-09T07:47:35.157Z",
        "deleted_at": null
      },
      "subtotal": 10,
      "total": 10,
      "original_total": 10,
      "discount_total": 0,
      "discount_tax_total": 0,
      "tax_total": 0,
      "original_tax_total": 0,
      "raw_subtotal": {
        "value": "10",
        "precision": 20
      },
      "raw_total": {
        "value": "10",
        "precision": 20
      },
      "raw_original_total": {
        "value": "10",
        "precision": 20
      },
      "raw_discount_total": {
        "value": "0",
        "precision": 20
      },
      "raw_discount_tax_total": {
        "value": "0",
        "precision": 20
      },
      "raw_tax_total": {
        "value": "0",
        "precision": 20
      },
      "raw_original_tax_total": {
        "value": "0",
        "precision": 20
      }
    }
  ],
  "customer": {
    "id": "cus_01J7AW11PHCA62GP0NBYM7DJKB",
    "company_name": null,
    "first_name": null,
    "last_name": null,
    "email": "user@test.com",
    "phone": null,
    "has_account": false,
    "metadata": null,
    "created_at": "2024-09-09T07:47:30.130Z",
    "updated_at": "2024-09-09T07:47:30.130Z",
    "deleted_at": null,
    "created_by": null
  },
  "sales_channel": {
    "id": "sc_01J743P6PA41FZ5GF1EYVYFC9Q",
    "name": "Default Sales Channel",
    "description": "Created by Medusa",
    "is_disabled": false,
    "created_at": "2024-09-06T16:46:42.379Z",
    "metadata": null,
    "updated_at": "2024-09-06T16:46:42.379Z",
    "deleted_at": null
  },
  "fulfillments": [
    {
      "id": "ful_01J7AW26XKSSVRXXN22A05G6R3",
      "location_id": "sloc_01J743P6TWKG0P3R0AFS4EAH4D",
      "packed_at": "2024-09-09T07:48:08.241Z",
      "shipped_at": "2024-09-09T07:48:17.226Z",
      "marked_shipped_by": null,
      "created_by": null,
      "delivered_at": null,
      "canceled_at": null,
      "data": {},
      "provider_id": "manual_manual",
      "shipping_option_id": "so_01J743P6WB5F22NJ59HAK8W3AW",
      "metadata": null,
      "shipping_option": {
        "id": "so_01J743P6WB5F22NJ59HAK8W3AW"
      },
      "delivery_address": {
        "id": "fuladdr_01J7AW26XKQ3Y2X6YSV34C489C"
      },
      "created_at": "2024-09-09T07:48:08.243Z",
      "updated_at": "2024-09-09T07:48:17.238Z",
      "deleted_at": null,
      "provider": {
        "id": "manual_manual"
      },
      "items": [
        {
          "id": "fulit_01J7AW26XK1MQGNX5H85W99Q6S",
          "title": "M",
          "sku": "SHORTS-M",
          "barcode": "",
          "raw_quantity": {
            "value": "1",
            "precision": 20
          },
          "line_item_id": "ordli_01J7AW16KN9DACFVRNV9QX1Z7K",
          "inventory_item_id": null,
          "fulfillment_id": "ful_01J7AW26XKSSVRXXN22A05G6R3",
          "created_at": "2024-09-09T07:48:08.244Z",
          "updated_at": "2024-09-09T07:48:17.239Z",
          "deleted_at": null,
          "quantity": 1
        }
      ],
      "labels": [
        {
          "id": "fulla_01J7AW2FPPPTNA3H88XCVTH9J5",
          "tracking_number": "11111111111111111111",
          "tracking_url": "#",
          "label_url": "#",
          "fulfillment_id": "ful_01J7AW26XKSSVRXXN22A05G6R3",
          "created_at": "2024-09-09T07:48:17.238Z",
          "updated_at": "2024-09-09T07:48:17.238Z",
          "deleted_at": null
        }
      ]
    }
  ],
  "payment_collections": [
    {
      "id": "pay_col_01J7AW15A5ZVQWERERT3JE16HS",
      "currency_code": "eur",
      "raw_amount": {
        "value": "20",
        "precision": 20
      },
      "raw_authorized_amount": {
        "value": "20",
        "precision": 20
      },
      "raw_captured_amount": {
        "value": "20",
        "precision": 20
      },
      "raw_refunded_amount": {
        "value": "0",
        "precision": 20
      },
      "region_id": "reg_01J743P6SN6V7P4E5BTS5B1E85",
      "created_at": "2024-09-09T07:47:33.829Z",
      "updated_at": "2024-09-09T07:48:02.326Z",
      "deleted_at": null,
      "completed_at": null,
      "status": "authorized",
      "metadata": null,
      "payments": [
        {
          "id": "pay_01J7AW16J779DY908X9H5CA32X",
          "raw_amount": {
            "value": "20",
            "precision": 20
          },
          "currency_code": "eur",
          "provider_id": "pp_system_default",
          "cart_id": null,
          "order_id": null,
          "customer_id": null,
          "data": {},
          "metadata": null,
          "created_at": "2024-09-09T07:47:35.112Z",
          "updated_at": "2024-09-09T07:48:02.315Z",
          "deleted_at": null,
          "captured_at": "2024-09-09T07:48:02.312Z",
          "canceled_at": null,
          "payment_collection_id": "pay_col_01J7AW15A5ZVQWERERT3JE16HS",
          "payment_session": {
            "id": "payses_01J7AW15AWZRWHD7VJ3PN15S15"
          },
          "refunds": [],
          "amount": 20
        }
      ],
      "amount": 20,
      "authorized_amount": 20,
      "captured_amount": 20,
      "refunded_amount": 0
    }
  ],
  "payment_status": "captured",
  "fulfillment_status": "shipped"
}
```

### Screenshots
When return back from order detail to order list, the seems like this
![Screenshot from 2024-09-09 08-49-18](https://github.com/user-attachments/assets/7986d80c-bd55-4468-bf83-b222ca38b556)

### Code snippets

If applicable, add code samples to help explain your problem

### Additional context

Add any other context about the problem here

**Comment by fPolic** [comment]
hey @ricardolopesit, thanks for the report!

I couldn't reproduce the issue on my end. Could you share more details, what preview version are you running?

**Comment by ricardolopesdev** [comment]
> hey @ricardolopesit, thanks for the report!
> 
> I couldn't reproduce the issue on my end. Could you share more details, what preview version are you running?

I'am using this ones 
![image](https://github.com/user-attachments/assets/146453f8-6794-488e-8e7e-dc42c17a997b)

**Comment by ricardolopesdev** [comment]
Hello.
When change the payment status don't refresh the list to correct state of payment only change when  the fulfilment status change like the the fix #9076
